### DISCUSSION TITLE
How to configure tailwind.config.js to check files from another wordpress plugin

### DISCUSSION THREAD

**Discussion by online-lorno** [wind-press/windpress#6] [General]
I created a tailwind.config.js file, along with the default main.css. How can I modify the content to check files from a different plugin to generate the build css file?

```javascript
export default {
  content: [
    "./**/*.php",
    "./**/*.js", 
    "../wp-plugin-name/*.php",
    "../wp-plugin-name/templates/*.php",
    "../wp-plugin-name/assets/**/*.js",
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}
```

Currently this does not work in which some of the js files has a functionality to append html strings to an element. Those html strings contain tailwindcss classes, but they're not included in the build file.

**Comment by online-lorno** [comment]
Update, the php files from the plugin are being checked, but the js files containing code like:

```javascript
const template = document.createElement("template");
template.innerHtml = `
    <div class="h-9 p-6">content</div>
`;
```

are not working. `h-9` and `p-6` are not generated in the output file of windpress.

**Comment by suasgn** [comment]
You can create a custom scanner.  

https://wind.press/docs/integrations/custom-theme#production-environment

**Comment by suasgn** [comment]
The TW4 provides an easy method for scanning additional files.

https://wind.press/docs/guide/configuration/tw-4/file-main-css#wp-content-directory
